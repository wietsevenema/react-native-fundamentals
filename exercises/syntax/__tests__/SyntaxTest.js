/* @flow */
import React from 'react';
import { Classic, Functional, Classy } from '../Syntax';
import renderer from 'react-test-renderer';

describe('Syntax', () => {
  it('with classic', () => {
    const tree = renderer.create(
      <Classic />
    );
    expect(tree).toMatchSnapshot();
  });

  it('with classy', () => {
    const tree = renderer.create(
      <Functional />
    );
    expect(tree).toMatchSnapshot();
  });

  it('with classy', () => {
    const tree = renderer.create(
      <Classy />
    );
    expect(tree).toMatchSnapshot();
  });

});
