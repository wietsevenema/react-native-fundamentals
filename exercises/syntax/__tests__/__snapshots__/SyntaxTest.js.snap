// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Syntax renders classic 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Hello world!
  </Text>
</View>
`;

exports[`Syntax renders classy 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Hello world!
  </Text>
</View>
`;

exports[`Syntax renders classy 2`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Hello world!
  </Text>
</View>
`;
